rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -32,8 +32,8 @@ paths:
     parameters:
       - $ref: '#/components/parameters/AcceptVersion'
     post:
-      summary: Transformation request
-      description: A request to transform a file into discovery metadata and other supporting products.
+      summary: Submit transformation request
+      description: A request to transform an intel resource into discovery metadata and other supporting products.
```

Comment:
I agree with the first part that we are not transforming an intel, we are transforming **for** a given intel. I disagree with the last part that it is just a resource that we are transforming since we are seeing that they might be case where we will use a lot more data than the resource to perform the transformation. This is why we should keep it in relation to the intel for which we are performing the transformation.

Reply:
I understand resource to be just a file, not the product or original file.

Reply:
We might as well start thinking about intel since we will need to start correlating these requests to the actual intel with the intel id (a.k.a. as URI) such that troubleshooting is easier between the various components of the system.